Tasks/AzureFileCopyV4/AzureFileCopyRemoteJob.ps1

```diff
@@ -67,7 +67,7 @@ $AzureFileCopyRemoteJob = {
 
                 # Downloading AzCopy from URL and copying it in $azcopyZipPath
                 $webclient = New-Object System.Net.WebClient
-                $webclient.DownloadFile('https://vstsagenttools.blob.core.windows.net/tools/azcopy/10.3/AzCopy.zip',$azCopyZipPath)
+                $webclient.DownloadFile('https://vstsagenttools.blob.core.windows.net/tools/azcopy/10.8/AzCopy.zip',$azCopyZipPath)
```

Comment:
In this file, change version to 10.8.0 at line 86

Reply:
done